cli/ballerina-cli/src/test/resources/test-resources/projectsForDumpGraph/package_c/modules/mod_c2/mod2.bal

```diff
@@ -0,0 +1,2 @@
+public function func2() {
+}
```

Comment:
this file needs a blank line at the end of the file

Reply:
fixed